docs/Training-on-Amazon-SageMaker-RL.md

```diff
@@ -0,0 +1,283 @@
+
+### Training the model
+
+After setting up the configuration and model customization, we’re ready to start the SageMaker RL training job. See the following code:
+
+
+```
+metric_definitions = RLEstimator.default_metric_definitions(RLToolkit.RAY)
+
+estimator = RLEstimator(entry_point="train-unity.py",
+                        source_dir='src',
+                        dependencies=["common/sagemaker_rl"],
+                        image_name=custom_image_name,
+                        role=role,
+                        train_instance_type=instance_type,
+                        train_instance_count=1,
+                        output_path=s3_output_path,
+                        base_job_name=job_name_prefix,
+                        metric_definitions=metric_definitions,
+                        hyperparameters={
+                # customize Ray parameters here
+                        }
+                    )
+s3_binary_path = '<s3 path for your Unity files> e.g. s3://bucket/unity-data'
+estimator.fit({'train': s3_binary_path}, wait=local_mode)
+job_name = estimator.latest_training_job.job_name
+print("Training job: %s" % job_name)
+```
+
+Inside the code, we specify a few parameters:
+
+* `entry_point` – The path to the training script we wrote that specifies the training process
+* `source_dir` – The path to the directory with other training source code dependencies aside from the entry point file
+* `dependencies` – A list of paths to directories with additional libraries to be exported to the container
+
+In addition, we state the container image name, training instance information, output path, and selected metrics. We are also allowed to customize any Ray-related parameters using the hyperparameters argument. We launch the SageMaker RL training job by calling `estimator.fit`, and start the model training process based on the specifications in the training script.
+At a high level, the training job initiates a neural network and updates the network gradually towards the direction in which the agent collects higher reward. Through multiple trials, the agent eventually learns how to navigate to the high-rewarding location efficiently. SageMaker RL handles the entire process and allows you to view the training job status in the Training jobs page on the SageMaker console.
+
+### Evaluating the model
+
+When model training is complete, we can load the trained model to evaluate its performance. Similar to the setup in the training script, we wrap the Unity environment with a gym wrapper. We then create an agent by loading the trained model.
+To evaluate the model, we run the trained agent multiple times against the environment with a fixed agent and target initializations, and add up the cumulative rewards the agent collects at each step for each episode.
+Out of five episodes, the average episode reward is 0.92 with the maximum reward of 0.93 and minimum reward of 0.89, suggesting the trained model indeed performs well.
```

Comment:
this seems out of place.  I would put a qualifier like "For example, out of the five episodes..."